packages/site/src/ui/page-heading.js

```diff
@@ -24,19 +29,21 @@ const ChangeLogContainer = props =>
       }
     `}</style>
   </div>
+)
 
 const renderChangeLog = ({ packageName }) =>
-  packageName
-    ? <ChangeLogContainer>
-        <ChangeLog packageName={packageName} />
-      </ChangeLogContainer>
-    : null
+  packageName ? (
+    <ChangeLogContainer>
+      <ChangeLog packageName={packageName} />
+    </ChangeLogContainer>
+  ) : null
 
-export default props =>
+export default props => (
```

Comment:
I like the good ol days where prettier didn't want (parens) around these.

Reply:
i can look into that!

Reply:
No, wasn't a thing to look into, just something that prettier changed in the earlier days. I don't think there's anything to do here.